application-forum-ui/src/main/resources/ForumCode/ConfigurationAdmin.xml

```diff
@@ -40,20 +40,22 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{include reference="Licenses.Code.VelocityMacros"/}}
-
+{{include reference='ForumCode.Macros'/}}
 {{velocity}}
 #set ($mainReference = $services.model.createDocumentReference('', 'ForumCode', 'ConfigurationClass'))
 #if (!$services.licensing.licensor.hasLicensureForEntity($mainReference))
 
   {{error}}#getMissingLicenseMessage('forum.extension.name'){{/error}}
 
 #else
-  {{include reference="ForumCode.FlagsHome"/}}
-
+  {{include reference='ForumCode.FlagsHome'/}}
   == $services.localization.render('forum.administration.configuration.title') ==
+  #checkIfHostNameIsValid($doc.getExternalURL(), $result)
```

Comment:
``checkIfHostNameIsValid`` is a bit misleading because ``localhost`` is a valid host. So either you call it ``checkIfHostNameIsValidForXXX`` (e.g. checkIfHostNameIsValidForShare) or you indicate what you check as in ``canShareURL`` (which I prefer) or ``checkIfURLCanBeShared``.